src/test/java/emissary/core/channels/FileChannelFactoryTest.java

```diff
@@ -0,0 +1,96 @@
+package emissary.core.channels;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
No hard rule but taking advantage of jupiter for unit tests in new classes would probably be helpful to reduce future transitions.